src/utility.ts

```diff
@@ -0,0 +1,15 @@
+export const existsSync = (filename: string): boolean => {
+  try {
+    Deno.statSync(filename);
+    // successful, file or directory must exist
+    return true;
+  } catch (error) {
+    if (error instanceof Deno.errors.NotFound) {
+      // file or directory does not exist
+      return false;
+    } else {
+      // unexpected error, maybe permissions, pass it along
+      throw error;
+    }
+  }
+};
```

Comment:
It's actually pretty easy to write *Promises* in Deno... I'm so used to Node that if I want to use a *Promise* I have to manually wrap their callbacks in a *Promise*. Well, everything in Deno that is not synchronous is already wrapped in a *Promise* 😀 
```ts
export const exists = async (filename: string): boolean => {
  try {
    await Deno.stat(filename);
    // successful, file or directory must exist
    return true;
  } catch (error) {
    if (error instanceof Deno.errors.NotFound) {
      // file or directory does not exist
      return false;
    } else {
      // unexpected error, maybe permissions, pass it along
      throw error;
    }
  }
};
```

Reply:
same as above, makes sense to add a async version 👍 https://github.com/drashland/sinco/pull/19/commits/72e6566107044e5f8f0741cab9d2aa87bd076afd